cockatrice/src/decklistmodel.cpp

```diff
@@ -300,8 +300,9 @@ QModelIndex DeckListModel::addCard(const QString &cardName, const QString &zoneN
 
 QModelIndex DeckListModel::nodeToIndex(AbstractDecklistNode *node) const
 {
-    if (node == root)
+    if (node == nullptr || node == root)
```

Comment:
Have a look at any code that calls this method too? See if the node they pass being null would break other things